src/toil/lib/aws/__init__.py

```diff
@@ -25,3 +25,12 @@ def client(*args, **kwargs):
 def resource(*args, **kwargs):
     import boto3
     return boto3.resource(*args, **kwargs)
+
+
+def zone_to_region(zone: str):
```

Comment:
This isn't really a new function, but if we're touching it it couldn't hurt to give it a return type hint.

Reply:
Fixed.